File: Lib/Telerik/Examples/QuickStart/Tools/ThemeViewer/LaunchForm.cs
using Telerik.QuickStart.WinControls;

namespace Telerik.Examples.WinControls.Tools.ThemeViewer
{
    public partial class LaunchForm : ExamplesLauncherForm, ICustomThemeExamplesForm
    {
        public LaunchForm()
        {
            InitializeComponent();

            this.pictureBoxLaunchExample.ButtonImage = Telerik.Examples.WinControls.Properties.Resources.launch;
        }
    }
}

File: Lib/Telerik/Examples/QuickStart/Tools/ThemeViewer/LaunchForm.vb
Imports Telerik.QuickStart.WinControls

Namespace Telerik.Examples.WinControls.Tools.ThemeViewer
	Partial Public Class LaunchForm
		Inherits ExamplesLauncherForm
		Implements ICustomThemeExamplesForm
		Public Sub New()
			InitializeComponent()

			Me.pictureBoxLaunchExample.ButtonImage = My.Resources.launch
		End Sub
	End Class
End Namespace
